error[E0658]: use of unstable library feature 'stdarch_arm_crc32'
  --> ../../third_party/rust/chromium_crates_io/vendor/crc32fast-1.4.2/src/specialized/aarch64.rs:56:47
   |
56 |     c32 = pre_quad.iter().fold(c32, |acc, &b| arch::__crc32b(acc, b));
   |                                               ^^^^^^^^^^^^^^
   |
   = note: see issue #117215 <https://github.com/rust-lang/rust/issues/117215> for more information
   = help: add `#![feature(stdarch_arm_crc32)]` to the crate attributes to enable


This feature arrives with rustc 1.80, so we can drop this once rustc-web
gets updated.

--- a/third_party/rust/chromium_crates_io/vendor/crc32fast-1.4.2/src/lib.rs
+++ b/third_party/rust/chromium_crates_io/vendor/crc32fast-1.4.2/src/lib.rs
@@ -35,6 +35,7 @@
 //! optimal implementation for the current CPU feature set.
 
 #![cfg_attr(not(feature = "std"), no_std)]
+#![cfg_attr(target_arch = "aarch64", feature(stdarch_arm_crc32))]
 
 #[deny(missing_docs)]
 #[cfg(test)]
--- a/third_party/rust/crc32fast/v1/BUILD.gn
+++ b/third_party/rust/crc32fast/v1/BUILD.gn
@@ -35,6 +35,7 @@ cargo_crate("lib") {
       "Fast, SIMD-accelerated CRC32 (IEEE) checksum computation"
   library_configs -= [ "//build/config/compiler:chromium_code" ]
   library_configs += [ "//build/config/compiler:no_chromium_code" ]
+  library_configs -= [ "//build/config/compiler:disallow_unstable_features" ]
   executable_configs -= [ "//build/config/compiler:chromium_code" ]
   executable_configs += [ "//build/config/compiler:no_chromium_code" ]
   proc_macro_configs -= [ "//build/config/compiler:chromium_code" ]
